test/test-functions

```diff
@@ -11,6 +11,8 @@ echo "TESTDIR=\"$TESTDIR\"" > .testdir${TEST_RUN_ID:+-$TEST_RUN_ID}
 export TESTDIR
 
 KVERSION=${KVERSION-$(uname -r)}
+DRACUT=${DRACUT-${basedir}/dracut.sh}
```

Comment:
Here is the successful test for the first four test cases: https://salsa.debian.org/debian/dracut/-/merge_requests/24

You can either set `DRACUT=dracut` or `DRACUT=/usr/bin/dracut` to use an actual distro installed dracut.

The CI broke because of a trivial forward porting failure (`basedir` needs to be defined before usage).